src/View/Widgets/AppMenu.vala

```diff
@@ -112,6 +113,21 @@ public class Files.AppMenu : Gtk.Popover {
         };
         show_remote_thumbnails.get_style_context ().add_class (Gtk.STYLE_CLASS_MENUITEM);
 
+        ///TRANSLATORS The format of the date (possibly with time) shown in the Modified column of the file view
+        var datetimeformat_label = new Gtk.Label (_("Date & Time Format"));
+        datetimeformat_combo = new Gtk.ComboBoxText ();
+        datetimeformat_combo.append_text (DateFormatMode.ISO.to_string ());
+        datetimeformat_combo.append_text (DateFormatMode.LOCALE.to_string ());
+        datetimeformat_combo.append_text (DateFormatMode.INFORMAL.to_string ());
+        //TODO Add a custom format wizard? Or a detailed informat format? Or "days ago" format?
+        datetimeformat_combo.active = (int) Files.Preferences.get_default ().date_format;
```

Comment:
I'm not sure if a combobox is appropriate because we're already opening an popover here so resulting a duplicated popover.

What do you think about using a radio button like we do in Maps?

<img width="229" height="281" alt="スクリーンショット 2025-09-18 20 55 35" src="https://github.com/user-attachments/assets/976c05cc-38e8-4f62-8c10-b6d6abf47e17" />

Reply:
Thanks - I'll try that.